internal/gps/vcs_source_test.go

```diff
@@ -520,12 +522,16 @@ func testHgSourceInteractions(t *testing.T) {
 	<-donech
 }
 
-// Fail a test if the specified binaries aren't installed.
+// Skip a test if the specified binaries aren't installed.
 func requiresBins(t *testing.T, bins ...string) {
 	for _, b := range bins {
 		_, err := exec.LookPath(b)
 		if err != nil {
-			t.Fatalf("%s is not installed", b)
+			if *Strict {
+				t.Fatalf("%s is not installed", b)
+			} else {
+				t.Skipf("Skipping: %s is not installed", b)
```

Comment:
Let's also note the presence of the flag in the printed message, so that the user has the full context on how this feature works.